docs/admin/performance_tuning.md

```diff
@@ -1,229 +1,165 @@
 # DAOS Performance Tuning
 
+This section aims at documenting how to validate the performance of the baseline
+building blocks in a DAOS system (i.e. network and storage) and then the full
+stack.
+
 ## Network Performance
 
 The DAOS [CaRT][1] layer can validate and benchmark network communications in
 the same context as an application and using the same networks/tuning options
 as regular DAOS.
 
 The CaRT `self_test` can run against the DAOS servers in a production environment
-in a non-destructive manner. CaRT `self_test` supports different message sizes,
-bulk transfers, multiple targets, and the following test scenarios:
+in a non-destructive manner. The only requirement is to have the DAOS agent
+running on the client node where self\_test is run.
```

Comment:
and daos_server running on the server(s)?  Does the storage need to be formatted?

Reply:
fixed.